### PR TITLE
Prompt with new vpcs, fixes: #1083

### PR DESCRIPTION
**Pull request by ziajka**

### CODE REVIEW

**Comment by ziajka** [comment]
Seems to fail on Windows. Let me check it first.

File: requirements.txt [hunk 1]
```diff
@@ -7,3 +7,4 @@ raven>=5.23.0
 psutil>=3.0.0
 zipstream>=1.1.4
 typing>=3.5.3.0 # Otherwise yarl fail with python 3.4
+prompt-toolkit
```
Comment (noplay) [inline requirements.txt]: Do we need a minimal version?

File: gns3server/utils/asyncio/embed_shell.py [hunk 2]
```diff
@@ -111,29 +127,130 @@ def run(self):
...
+        self._loop = loop
+        self._cli = None
+        self.encoding = 'utf-8'
+
+    @asyncio.coroutine
+    def connected(self):
+        self._cli = CommandLineInterface(
+            application=create_prompt_application(self._shell.prompt),
+            eventloop=UnstoppableEventLoop(create_asyncio_eventloop(self._loop)),
+            output=Vt100_Output(self, lambda: Size(rows=40, columns=79)))
```
Comment (noplay) [inline gns3server/utils/asyncio/embed_shell.py]: It's bug if the terminal is smaller. I think we can get the size via telnet

File: requirements.txt (hunk 1 above)
Comment (ziajka) [inline requirements.txt]: I don't think so, actually cannot see any backlog of versions (to choose the minimal).

**Comment by ziajka** [comment]
@noplay done :)

**Comment by noplay** [comment]
Bravo it's work !!!

**Pull request merged**